scalafix-tests/input/src/main/scala/test/DisableRegex.scala

```diff
@@ -0,0 +1,49 @@
+/*
+rules = Disable
+Disable.symbols = [
+  {
+    regex = "scala.collection.mutable.*"
+    message = "use immutable structures"
+  }
+]
+
+Disable.unlessInside = [
+  {
+    safeBlock = "test.DisableRegex.IO"
+    symbols = [
+      {
+        regex = {
+          includes = "java.io.*"
+          excludes = "java.io.InputStream"
+        }
+        message = "input/output are only allowed within IO block"
+      }
+    ]
+  }
+]
+ */
+package test
+
+object DisableRegex {
+  import scala.collection.mutable._ // ok
+  import scala.collection.mutable   // ok
+
+  @SuppressWarnings(Array("Disable.ListBuffer"))
+  @SuppressWarnings(Array("Disable.<init>"))
+  val buffer = new ListBuffer[Int]()
+
+  @SuppressWarnings(Array("Disable.mutable"))
+  @SuppressWarnings(Array("Disable.HashMap"))
+  @SuppressWarnings(Array("Disable.<init>"))
+  val abc = new mutable.HashMap[String, String]()
```

Comment:
Intuitively, there is only one symbol being referenced here `HashMap.<init>` so I think it's a bit weird that three errors are reported.

I think it would make sense to skip the qualifier of a `Term.Select` if it only contains `Term.Select` or `Term.Name`.

```scala
a.b.C // only report on C
d.E // only report on E
f.g(1).h // report on h and g
```

It should be simple to do this check with something like
```scala
def skipQualifier(qual: Term) Boolean = qual match {
  case _: Term.Name => true
  case Term.Select(q, _) => skipQualifier(q)
}
```